libs/dh/admin/data-access-api/src/lib/dh-admin-create-user-role-management-data-access-api.store.ts

```diff
@@ -57,16 +54,16 @@ export class DhAdminCreateUserRoleManagementDataAccessApiStore extends Component
     super(initialState);
   }
 
-  private readonly getSelectablePermissions = this.effect(
-    (trigger$: Observable<void>) =>
+  public readonly getSelectablePermissions = this.effect(
+    (trigger$: Observable<EicFunction>) =>
       trigger$.pipe(
-        switchMap(() =>
+        switchMap((x) =>
```

Comment:
Suggestion: use variable name that gives context

Reply:
Fixed